tests/test_cli_compile.py

```diff
@@ -2973,3 +2973,17 @@ def test_config_option(pip_conf, runner, tmp_path, make_config_file):
 
     assert out.exit_code == 0
     assert "Dry-run, so nothing updated" in out.stderr
+
+
+def test_no_config_option(pip_conf, runner, tmp_path, make_config_file):
```

Comment:
```suggestion
def test_no_config_option_overrides_config_with_defaults(pip_conf, runner, tmp_path, make_config_file):
```
Would this be more accurate?